pkgs/applications/office/foliate/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, lib, fetchFromGitHub, meson, gettext, glib, gjs, ninja, python3, gtk3
+, webkitgtk, gsettings-desktop-schemas, wrapGAppsHook, desktop-file-utils
+, gobject-introspection }:
+
+stdenv.mkDerivation rec {
+  pname = "foliate";
+  version = "2.6.3";
+
+  src = fetchFromGitHub {
+    owner = "johnfactotum";
+    repo = pname;
+    rev = version;
+    sha256 = "0ribqaxl8g1i83fxbn288afwbzzls48ni57xqi07d19p9ka892mr";
+  };
+
+  nativeBuildInputs = [ meson ninja python3 wrapGAppsHook ];
+
+  postPatch = ''
+    patchShebangs build-aux/meson/postinstall.py
+  '';
+
+  postFixup = ''
+    echo "fixing wrapper"
```

Comment:
It is not good to pollute the stdout with those messages.

I love Gutenberg debugging, but it should not to be in production code.

Reply:
Sorry for the late fix ;D https://github.com/NixOS/nixpkgs/pull/167977